Transcelerator/TransceleratorTests/TransceleratorTests.csproj

```diff
@@ -125,6 +224,12 @@
     </Reference>
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
+    <Reference Include="taglib-sharp, Version=2.2.0.0, Culture=neutral, PublicKeyToken=db62eba44689b5b0, processorArchitecture=MSIL">

```

Comment:
Do we need both TagLibSharp-patched and TagLibSharp? (IIRC all changes from -patched got accepted in TagLibSharp)